application/helpers/admin/statistics_helper.php

```diff
@@ -2383,10 +2383,11 @@ protected function displaySimpleResults($outputs, $results, $rt, $outputType, $s
     * @param mixed $surveyid
     * @param mixed $sql
     * @param mixed $usegraph
+    * @param mixed $ansID
      *
      *
     */
-    protected function displayResults($outputs, $results, $rt, $outputType, $surveyid, $sql, $usegraph, $browse, $sLanguage)
+    protected function displayResults($outputs, $results, $rt, $outputType, $surveyid, $sql, $usegraph, $ansID=1, $browse, $sLanguage)
```

Comment:
I am sceptical about adding yet another argument to an already too big function signature.